packages/vscode-extension/src/extension.ts

```diff
@@ -430,6 +441,33 @@ function registerChatParticipant(context: vscode.ExtensionContext) {
   context.subscriptions.push(generateManifestGUID);
 }
 
+/**
+ * Copilot Chat Participant for Office Add-in
+ */
+function registerOfficeChatParticipant(context: vscode.ExtensionContext) {
+  const participant = vscode.chat.createChatParticipant(
+    officeChatParticipantId,
+    officeChatRequestHandler
+  );
+  participant.iconPath = vscode.Uri.joinPath(context.extensionUri, "media", "teams.png");
```

Comment:
Just to check that we are using the same icon, right?

Reply:
We will use the M365 icon for Office agent. Will update in the next iteraion.
![image](https://github.com/OfficeDev/TeamsFx/assets/55721214/b3be53cd-8866-4797-bb28-988ff499f59c)